course/templates/course/index.html

```diff
@@ -12,81 +12,48 @@ <h1>{{ welcome_text|safe }}</h1>
 </div>
 {% endif %}
 
-<div class="row course-thumbnails">
-    {% site_advert as advert %}
-    {% if advert %}
-        <div class="col-sm-6 col-md-4 col-lg-3">
-            <div class="thumbnail advert-thumbnail">
-                <div class="image" aria-hidden="true">
-                    {% if advert.image %}
-                        <img src="{{ advert.image }}" alt="">
-                    {% else %}
-                        <h1 class="glyphicon glyphicon-comment"></h1>
-                    {% endif %}
-                </div>
-                <div class="caption">
-                    <h3>{{ advert.title }}</h3>
-                    <p>{{ advert.text|safe }}</p>
-                    <div class="buttons caption">
-                        <a class="btn btn-default btn-block" role="button" href="{{ advert.href }}" target="_blank">
-                            <span class="glyphicon glyphicon-new-window" aria-hidden="true"></span>
-                            {% trans "Read more" %}
-                        </a>
-                    </div>
-                </div>
-            </div>
+{% if my_instances %}
+    <div class="frontpage-panel">
+        <h2 class="panel-heading">{% trans "My courses"%}</h2>
+        <div class="row frontpage">
+            {% include "course/course_cards.html" with instances=my_instances %}
```

Comment:
Some parts of the A+ use template naming similar to how Ruby on Rails handles partials. I.e. this included file would be named `course/_course_cards.html` to indicate that it's supposed to be included from another template.

:thinking:  Something for me to include in the stuleguide..

Reply:
Oh, I didn't know that. But can definitely change it!

Reply:
Yeah. I don't think it's so common with Django, but it does help when listing templates in the filesystem and to not make mistakes when using the template filenames in the python code (as underscores shouldn't appear there).